packages/backend-for-frontend/src/utilities/errorMappers.ts

```diff
@@ -207,3 +206,57 @@ const delegationNotFoundErrorMapper = (error: ApiError<ErrorCodes>): number =>
 
 export const getDelegationByIdErrorMapper = delegationNotFoundErrorMapper;
 export const getDelegationsErrorMapper = delegationNotFoundErrorMapper;
+
+export const createEServiceTemplateVersionErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("noVersionInEServiceTemplate", () => HTTP_STATUS_BAD_REQUEST)
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const bffGetEServiceTemplateErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("eserviceTemplateVersionNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const bffGetCatalogEServiceTemplateErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with(
+      "tenantNotFound",
+      "catalogEServiceTemplatePublishedVersionNotFound",
+      () => HTTP_STATUS_NOT_FOUND
+    )
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const addEServiceInterfceByTemplateErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with(
+      "eserviceTemplateIsNotPublished",
+      "eserviceIsNotDraft",
+      "eserviceTemplateInterfaceDataNotValid",
+      "invalidInterfaceContentTypeDetected",
+      "invalidInterfaceFileDetected",
+      () => HTTP_STATUS_BAD_REQUEST
+    )
+    .with("invalidEserviceRequester", () => HTTP_STATUS_UNAUTHORIZED)
+    .with(
+      "templateDataNotFound",
+      "eserviceTemplateInterfaceNotFound",
+      "eserviceTemplateNotFound",
+      "eserviceTemplateVersionNotFound",
+      "eserviceDescriptorDraftNotFound",
+      "interfaceExtractingInfoError",
```

Comment:
this is a bad request

Reply:
handled in this [PR](https://github.com/pagopa/interop-be-monorepo/pull/1629)